scripts/npm-access.js

```diff
@@ -6,6 +6,14 @@ const getPkgPermissions = userOrOrg => {
   return JSON.parse(stringResponse);
 };
 
+const getMonorepoPackages = () => {
+  // get list of monorepo packages and remove lerna branding from output
+  const pkgs = execSync('$(yarn bin)/lerna ls | grep -v "^lerna"')
```

Comment:
should probably remove the `$(yarn bin)` since the script will have the correct path to resolve lerna

Reply:
will this work on windows?

Reply:
Nope, the filtering should be done in node space.